stressor/README.md

```diff
@@ -16,9 +16,10 @@ It works sequentially through the list of test plans, completing the runs for on
 ## Running
 
 1. It is prefered for you to run the stress test against your own personal "non-fork" of this repo (create a personal repo and push to it instead of using "fork" so it isn't part of the "network") to limit the number of action runs against the main branch.
-2. Update the stress-test.mts file `owner`, `repo`, and `defaultBranch` definitions near the top, as well as setting up the tests / matrix you want to test.
-3. Run it with `npm run --silent stress-test | tee some-output-file.md`.
-4. Running the script can take a while, as it is constrained by GitHub Actions availability and speed.
+2. Run it with the following command (replacing `myUser` and `myRepo` with the owner and name of your repository):
+
+      $ npx tsx stress-test.mts --owner myUser --repo myRepo
```

Comment:
i think we still need the ` | tee some-output-file.md` added on to here, or for that functionality to be built directly into the script. Right now, the script outputs the md contents to stdout when the script finishes, so we need to redirect that output to a file, or at least give people the option to. (The output is very long if you run it with all the test plans.)

Reply:
@outofambit it looks like the output only gets written if the script succeeds, correct? I've yet to get a successful full test plan run (either on my local or on codespaces) and haven't seen any output written out to a file yet... 

In general, I'm thinking we could use a more robust way of collecting the output from a run, whether it succeeds or not...

Reply:
@outofambit Totally

@ChrisC That sounds like a job for a separate patch. Let me know if you feel otherwise!